### ISSUE TITLE
Combined ES + GS data/ short and longreads in a single project

### ISSUE THREAD

**Issue by lynnpais** [broadinstitute/seqr#4024]
Combined ES + GS data/ short and longreads in a single project

**Comment by lynnpais** [comment]
Implement before PacBio longread data

**Comment by hanars** [comment]
Notes on search result behavior: https://docs.google.com/presentation/d/1hqDV8ulhviUcR5C4PtNUqkCLXKDsc6pccgFVlFmWUAU/edit#slide=id.p

**Comment by Mo7ammedFarahat** [comment]
Does that means that seqr works now with LR VCF? from PacBio and ONT?

**Comment by hanars** [comment]
> Does that means that seqr works now with LR VCF? from PacBio and ONT?

No, we have planned work for early 2025 to support PacBio, which is blocked by this work

**Comment by Mo7ammedFarahat** [comment]
And what about ONT? And tertiary analysis?

**Comment by hanars** [comment]
Those are not on our near-term roadmap, but as an open source project we are always willing to collaborate with other groups who are willing to contribute their developer time to build out seqr functionality

**Comment by jklugherz** [comment]
done!
